app/presenters/default_deadline_date_time.rb

```diff
@@ -0,0 +1,13 @@
+# used for computing the default deadline with an offset
```

Comment:
we aren't doing class-level comments throughout this codebase. you think we need one here?